src/peep/peep.c

```diff
@@ -2290,8 +2290,9 @@ void peep_update_ride_sub_state_7(rct_peep* peep){
 		vehicle = GET_VEHICLE(vehicle->next_vehicle_on_train);
 	}
 
-	// Unsure why backward rotation is missing.
-	if (ride->mode != RIDE_MODE_FORWARD_ROTATION){
+	// Check if ride is NOT Farris Wheel.
+	if (ride->mode != RIDE_MODE_FORWARD_ROTATION &&
+	        ride->mode != RIDE_MODE_BACKWARD_ROTATION){
 		if (vehicle->num_peeps - 1 != peep->current_seat)
```

Comment:
Any idea what this is targeting? I was trying to add a comment but I wasn't sure what it was trying to do.

As I'm not sure what other rides this accounts for I'm assuming all. It seems it would only run when the peep is the last person in the vehicle. Great or one person vehicles I guess